lib/matplotlib/animation.py

```diff
@@ -1147,13 +1291,38 @@ def _loop_delay(self, *args):
 
 
 class ArtistAnimation(TimedAnimation):
-    '''
-    Before calling this function, all plotting should have taken place
+    '''Animation using a fixed set of `Artist` objects.
+
+    Before creating an instance, all plotting should have taken place
     and the relevant artists saved.
 
-    *artists* is a list, with each list entry a collection of artists that
-    represent what needs to be enabled on each frame. These will be disabled
-    for other frames.
+
+    Parameters
+    ----------
+    fig : matplotlib.figure.Figure
+       The figure object that is used to get draw, resize, and any
+       other needed events.
+
+    artists : list
+        with each list entry a collection of artists that
+        represent what needs to be enabled on each frame. These will
+        be disabled for other frames.
+
+    interval : number, optional
```

Comment:
Broader question: Do we need to repeat the documentation of attributes from the base class?